types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
@amritk ah, that make sense, but I believe we should came with some comprehensive solution that is easier to use and maintain. I would never thought to use typings, as I'm using angular-cli and just define types in tsconfig.json. 

If I could have a question @RyanCavanaugh is it possible to release @types/stripe and @types/stripe-v2 to satisfy need of having 2 versions alongside in package.json? For now it's just using hacks like @amritk describe, but I believe we should have comprehensive solution, if Stripe recommends to have 2 versions of Stripe on site. This will allow to use declaration merging of namespaces. Now it's just impossible to do it without 2 seperate packages.